Formula/g/graph-tool.rb

```diff
@@ -4,8 +4,8 @@ class GraphTool < Formula
   desc "Efficient network analysis for Python 3"
   homepage "https://graph-tool.skewed.de/"
   # TODO: Update build for matplotlib>=3.9.0 to use `--config-settings=setup-args=...` for system dependencies
```

Comment:
the dependencies need to be updated as in #172782

Reply:
Done.